api/services/SiteCreator.js

```diff
@@ -176,6 +176,7 @@ function createSiteFromSource({ siteParams, user }) {
 
 function createSite({ user, siteParams }) {
   const templateName = siteParams.template;
+
   const template = templateName && TemplateResolver.getTemplate(templateName);
```

Comment:
just to make 100% sure - these variables don't need to be updated, right?

Reply:
no, I just added a newline for readability